drivers/src/memory_layout.rs

```diff
@@ -29,27 +32,28 @@ pub const ROM_DATA_ORG: u32 = 0x50000000;
 pub const CFI_STATE_ORG: u32 = 0x500003E4; // size = 6 words
 pub const BOOT_STATUS_ORG: u32 = 0x500003FC;
 pub const MAN1_ORG: u32 = 0x50000400;
-pub const MAN2_ORG: u32 = 0x50001C00;
-pub const FHT_ORG: u32 = 0x50003400;
-pub const LDEVID_TBS_ORG: u32 = 0x50003C00;
-pub const FMCALIAS_TBS_ORG: u32 = 0x50004000;
-pub const RTALIAS_TBS_ORG: u32 = 0x50004400;
-pub const PCR_LOG_ORG: u32 = 0x50004800;
-pub const MEASUREMENT_LOG_ORG: u32 = 0x50004C00;
-pub const FUSE_LOG_ORG: u32 = 0x50005000;
-pub const DPE_ORG: u32 = 0x50005400;
-pub const PCR_RESET_COUNTER_ORG: u32 = 0x50006800;
-pub const AUTH_MAN_IMAGE_METADATA_LIST_ORG: u32 = 0x50006C00;
-pub const DATA_ORG: u32 = 0x50007000;
-
-pub const STACK_ORG: u32 = 0x5001A000;
-pub const ROM_STACK_ORG: u32 = 0x5001C000;
-
-pub const ESTACK_ORG: u32 = 0x5001F800;
-pub const ROM_ESTACK_ORG: u32 = 0x5001F800;
-
-pub const NSTACK_ORG: u32 = 0x5001FC00;
-pub const ROM_NSTACK_ORG: u32 = 0x5001FC00;
+pub const MAN2_ORG: u32 = MAN1_ORG + MAN1_SIZE;
+pub const FHT_ORG: u32 = MAN2_ORG + MAN2_SIZE;
+pub const IDEVID_MLDSA_PUB_KEY_ORG: u32 = FHT_ORG + FHT_SIZE;
+pub const LDEVID_TBS_ORG: u32 = IDEVID_MLDSA_PUB_KEY_ORG + IDEVID_MLDSA_PUB_KEY_MAX_SIZE;
+pub const FMCALIAS_TBS_ORG: u32 = LDEVID_TBS_ORG + LDEVID_TBS_SIZE;
+pub const RTALIAS_TBS_ORG: u32 = FMCALIAS_TBS_ORG + FMCALIAS_TBS_SIZE;
+pub const PCR_LOG_ORG: u32 = RTALIAS_TBS_ORG + RTALIAS_TBS_SIZE;
+pub const MEASUREMENT_LOG_ORG: u32 = PCR_LOG_ORG + PCR_LOG_SIZE;
+pub const FUSE_LOG_ORG: u32 = MEASUREMENT_LOG_ORG + MEASUREMENT_LOG_SIZE;
+pub const DPE_ORG: u32 = FUSE_LOG_ORG + FUSE_LOG_SIZE;
+pub const PCR_RESET_COUNTER_ORG: u32 = DPE_ORG + DPE_SIZE;
+pub const AUTH_MAN_IMAGE_METADATA_LIST_ORG: u32 = PCR_RESET_COUNTER_ORG + PCR_RESET_COUNTER_SIZE;
```

Comment:
(Not for this PR) Just as an FYI, we're planning to make an update in main to just have a `PERSISTENT_ORG` address and reserve a big chunk of memory. And then it will up to the PersistentData driver to decide how to allocate that memory. I think that will be easier to maintain than needing to update it both there and in this file.

Reply:
Ack.